src/pages/bobaekeum/Main/MainKeum.scss

```
@@ -1,2 +1,288 @@
+        }
+        .likeMount {
+          width: 100%;
+
+          margin-left: 8px;
+          margin-top: 5px;
+          font-size: 11px;
+          font-weight: bold;
+        }
+        .comments {
+          width: 100%;
+          display: flex;
+          margin-left: 8px;
+          .my_name {
+            margin-left: 0;
+            margin-top: 8px;
+            font-weight: bold;
+            font-size: 12px;
+            margin-right: 8px;
+          }
+          .myComment {
+            margin-left: 0px;
+            font-size: 10px;
+            padding-top: 10px;
+          }
+        }
+        .othersComments {
+          margin-left: 8px;
+          .thatgle {
+            display: flex;
+            .uName {
+              margin-left: 0;
+              margin-top: 8px;
+              font-weight: bold;
+              font-size: 12px;
+              margin-right: 8px;
+            }
+            .othersComment {
+              margin-left: 0px;
+              font-size: 10px;
+              padding-top: 10px;
+            }
+          }
+        }
+        .typingArea {
+          width: 100%;
+          border-top: 1px solid rgb(175, 175, 175);
+          .typingText {
+            width: 275px;
+            padding: 8px;
+            font-size: 10px;
+            border: none;
+            outline: none;
+          }
+          .post {
+            border: none;
+            background-color: rgb(255, 255, 255);
+            color: rgb(120, 175, 238);
+            font-size: 10px;
+            padding-right: 8px;
+            margin-left: -3px;
+            cursor: pointer;
+          }
+        }
+      }
+    }
+  }
+}
+
+/* aside */
+.main-right {
+  width: 180px;
+  float: right;
+  margin-top: 10px;
+  margin-left: 0;
+  padding-right: 10px;
+  height: 330px;
+  display: flex;
+  flex-direction: column;
+  font-weight: bold;
+  font-size: 12px;
+  .aboutUser {
+    display: flex;
+    width: 100%;
+    margin-bottom: 10px;
+    .my {
+      width: 25px;
+      height: 25px;
+      border-radius: 100%;
+      margin-left: 5px;
+      margin-top: 6px;
+      margin-right: 10px;
+    }
+    .mynamelayout {
+      margin-left: 0;
+      padding-top: 9px;
+      .my_name {
+        margin-right: 8px;
+      }
+      .myrealname {
+        font-weight: normal;
+        color: rgb(166, 166, 166);
+        font-size: 10.5px;
+      }
+    }
+  }
+  .aboutStory {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    border: 1px solid rgb(175, 175, 175);
+    border-radius: 2px;
+    margin-left: 5px;
+    background-color: white;
+    margin-bottom: 10px;
+    .storyShow {
+      display: flex;
+      width: 100%;
+      .story {
+        margin-left: 8px;
+        margin-top: 5px;
+        font-weight: normal;
+        color: rgb(166, 166, 166);
+        font-size: 10px;
+        font-weight: 600;
+      }
+      .showAll {
+        margin-right: 5px;
+        margin-top: 5px;
+        margin-left: auto;
```

margin, padding 등 좌표 정보와 함께 사용 가능한 속성은 한 줄로 정의 가능합니다!
```suggestion
        margin: 5px 5px 0 auto;
```

https://developer.mozilla.org/ko/docs/Web/CSS/margin 참고해주세요!